src/nfp_devlink.c

```diff
@@ -121,8 +123,12 @@ nfp_devlink_port_split(struct devlink *devlink, unsigned int port_index,
 		goto out;
 	}
 
-	ret = nfp_devlink_set_lanes(pf, eth_port.index,
-				    eth_port.port_lanes / count);
+	lanes = eth_port.port_lanes / count;
+	if(eth_port.lanes == 10) {
```

Comment:
Please run checkpatch.pl script from the Linux sources on your patches to check for style issues.